reference/toolstacks/php/drupal/redis-drupal-7.md

```diff
@@ -57,46 +91,50 @@ variables to match.
 
 ```bash
 [
-   "sites/all/modules/redis/redis.autoload.inc"
+   "sites/all/modules/contrib/redis/redis.autoload.inc"
 ]
 ```
 
 > **note**
 > Remember to tick the JSON Value box.
 
 > **note**
-> Use the actual path to your Redis module in case it is in a different location. For example: `sites/all/modules/contrib/redis`.
+> Use the actual path to your Redis module in case it is in a different location. For example: `sites/all/modules/redis`. The
```

Comment:
Actually Drush Make defaults to `contrib` only if you ask it to. But we do ask it to in our example (https://github.com/platformsh/platformsh-example-drupal/blob/7.x/project.make#L10).